taskcluster/requirements.in

```diff
@@ -1,5 +1,5 @@
 # For instructions on managing dependencies, see:
 # https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/bootstrap-taskgraph.html
 
-taskcluster-taskgraph >= 7.0.2
-mozilla-taskgraph >= 1.1.0
+mozilla-taskgraph==1.2.2
+taskcluster-taskgraph==7.3.0
```

Comment:
Do you want `==` or `>=`?

Reply:
I think we should keep `==` so dependabot won't try to update it..? Thoughts?